<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statwindow</class>
 <widget class="QDialog" name="Statwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1367</width>
    <height>946</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QFrame" name="fr_information">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="fr_port_info">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="lb_port_lb">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Выбранный порт:   </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_port_info">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&gt;port</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_selections">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QFrame" name="fr_input">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="lb_select_year">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Выберите год:       </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_select_month">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Выберите месяц:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="fr_labels">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QComboBox" name="cb_select_year">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>2016</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2017</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_select_month">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Январь</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Февраль</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Март</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Апрель</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Май</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Июнь</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Июль</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Август</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Сентябрь</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Октябрь</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ноябрь</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Декабрь</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>746</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="fr_date_info">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lb_date_label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Выбранные даты вылета</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gb_from">
           <property name="title">
            <string>От</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="lb_from_date">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>&gt;date1</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gb_till">
           <property name="title">
            <string>До</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="lb_till_date">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>&gt;date2</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_statistics">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_year">
      <attribute name="title">
       <string>Статистика за год</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="fr_y_diagram">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QChartView" name="viewport_year"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="fr_y_util">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="lb_y_info">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Информация по вылетам / прилётам</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="te_y_info"/>
          </item>
          <item>
           <widget class="QPushButton" name="pb_paint_year">
            <property name="text">
             <string>Отобразить график</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_month">
      <attribute name="title">
       <string>Статистика за месяц</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QFrame" name="fr_m_diagram">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QChartView" name="viewport_month"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="fr_m_util">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="lb_m_info">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Информация по вылетам / прилётам</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="te_m_info"/>
          </item>
          <item>
           <widget class="QPushButton" name="pb_paint_month">
            <property name="text">
             <string>Отобразить график</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_controls">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>843</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_close">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ЗАКРЫТЬ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
